project_recalculate/README.rst

```diff
@@ -0,0 +1,100 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+===================
+Project Recalculate
+===================
+
+This module recalculates tasks' start/end dates depending on project's
```

Comment:
(nitpick) -> `+This module recalculates Task start/end dates depending on Project`